tiktoken-rs/src/vendor_tiktoken.rs

```diff
@@ -541,6 +553,20 @@ impl CoreBPE {
             Err(e) => Err(anyhow!("Unable to decode into a valid UTF-8 string: {}", e)),
         }
     }
+
+    // tokenize a string and return the decoded tokens using the correct BPE model
+    // for example: "Hello world" -> ["Hello", " world"]
```

Comment:
Consider using a rust doc string here with `///` so that this information renders in the crate documentation. Rust doc is powerful because it can also run tests found in the documentation. See the doc string of other exposed functions here

Reply:
updated - let me know what you think

Reply:
Looks good!